crates/mun_codegen/src/snapshots/test__issue_225.snap

```diff
@@ -0,0 +1,75 @@
+---
+source: crates/mun_codegen/src/test.rs
+expression: "struct Num {\n    value: i64,\n}\n\npub fn foo(b: i64) {\n    Num { value: b }.value;\n}\n\npub fn bar(b: i64) {\n    { let a = Num { value: b }; a}.value;\n}"
```

Comment:
This is from another PR, so maybe merge that one first and then rebase this PR

Reply:
Done